q2_diversity/_ordination.py

```diff
@@ -38,15 +38,17 @@ def tsne(distance_matrix: skbio.DistanceMatrix,
          perplexity: float = 25.0,
          n_iter: int = 1000,
          learning_rate: float = 200.0,
-         early_exaggeration: float = 12.0) -> skbio.OrdinationResults:
+         early_exaggeration: float = 12.0,
+         random_state: int = 1) -> skbio.OrdinationResults:
```

Comment:
This should default to `None` so that it's optional (it's also a valid input for sklearn with the expected semantics).